extension/tpch/dbgen/dbgen.cpp

```diff
@@ -22,77 +22,6 @@
-    {LINE, 675466456, 0, O_LCNT_MAX},          /* L_SMODE_SD   18 */
-    {LINE, 1808217256, 0, O_LCNT_MAX},         /* L_PKEY_SD    19 */
-    {LINE, 2095021727, 0, O_LCNT_MAX},         /* L_SKEY_SD    20 */
-    {LINE, 1769349045, 0, O_LCNT_MAX},         /* L_SDTE_SD    21 */
-    {LINE, 904914315, 0, O_LCNT_MAX},          /* L_CDTE_SD    22 */
-    {LINE, 373135028, 0, O_LCNT_MAX},          /* L_RDTE_SD    23 */
-    {LINE, 717419739, 0, O_LCNT_MAX},          /* L_RFLG_SD    24 */
-    {LINE, 1095462486, 0, O_LCNT_MAX * 2},     /* L_CMNT_SD    25 */
-    {CUST, 881155353, 0, 9},                   /* C_ADDR_SD    26 */
-    {CUST, 1489529863, 0, 1},                  /* C_NTRG_SD    27 */
-    {CUST, 1521138112, 0, 3},                  /* C_PHNE_SD    28 */
-    {CUST, 298370230, 0, 1},                   /* C_ABAL_SD    29 */
-    {CUST, 1140279430, 0, 1},                  /* C_MSEG_SD    30 */
-    {CUST, 1335826707, 0, 2},                  /* C_CMNT_SD    31 */
-    {SUPP, 706178559, 0, 9},                   /* S_ADDR_SD    32 */
-    {SUPP, 110356601, 0, 1},                   /* S_NTRG_SD    33 */
-    {SUPP, 884434366, 0, 3},                   /* S_PHNE_SD    34 */
-    {SUPP, 962338209, 0, 1},                   /* S_ABAL_SD    35 */
-    {SUPP, 1341315363, 0, 2},                  /* S_CMNT_SD    36 */
-    {PART, 709314158, 0, 92},                  /* P_NAME_SD    37 */
-    {ORDER, 591449447, 0, 1},                  /* O_PRIO_SD    38 */
-    {LINE, 431918286, 0, 1},                   /* HVAR_SD      39 */
-    {ORDER, 851767375, 0, 1},                  /* O_CKEY_SD    40 */
-    {NATION, 606179079, 0, 2},                 /* N_CMNT_SD    41 */
-    {REGION, 1500869201, 0, 2},                /* R_CMNT_SD    42 */
-    {ORDER, 1434868289, 0, 1},                 /* O_LCNT_SD    43 */
-    {SUPP, 263032577, 0, 1},                   /* BBB offset   44 */
-    {SUPP, 753643799, 0, 1},                   /* BBB type     45 */
-    {SUPP, 202794285, 0, 1},                   /* BBB comment  46 */
-    {SUPP, 715851524, 0, 1}                    /* BBB junk     47 */
-};
-double DBGenGlobals::dM = 2147483647.0;
-tdef DBGenGlobals::tdefs[10] = {
-    {"part.tbl", "part table", 200000, NULL, NULL, PSUPP, 0},
-    {"partsupp.tbl", "partsupplier table", 200000, NULL, NULL, NONE, 0},
-    {"supplier.tbl", "suppliers table", 10000, NULL, NULL, NONE, 0},
-    {"customer.tbl", "customers table", 150000, NULL, NULL, NONE, 0},
-    {"orders.tbl", "order table", 150000, NULL, NULL, LINE, 0},
-    {"lineitem.tbl", "lineitem table", 150000, NULL, NULL, NONE, 0},
-    {"orders.tbl", "orders/lineitem tables", 150000, NULL, NULL, LINE, 0},
-    {"part.tbl", "part/partsupplier tables", 200000, NULL, NULL, PSUPP, 0},
-    {"nation.tbl", "nation table", NATIONS_MAX, NULL, NULL, NONE, 0},
-    {"region.tbl", "region table", NATIONS_MAX, NULL, NULL, NONE, 0},
-};
-
-static seed_t *Seed = DBGenGlobals::Seed;
```

Comment:
not need to these variables anymore. Don't need backups as each DBGenContext will come with a fresh copy of these seeds.